database/drivers/postgres/parse.go

```diff
@@ -15,9 +15,23 @@ import (
 	"gnorm.org/gnorm/database/drivers/postgres/gnorm/tables"
 )
 
+// PG implements drivers.Driver interface for interacting with postgresql
+// database
+type PG struct {
```

Comment:
can you put this on one line? so `type PG struct{}` .  I find it easier to see that this is intentionally just an empty struct.  With the close brace on the second line, it takes me a second to figure out what's going on.